src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -199,10 +198,10 @@
     @Argument(fullName= StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName=StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             doc="The output recal file used by ApplyRecalibration", optional=false)
-    private String output;
+    private GATKPath output;
 
     @Argument(fullName="tranches-file", doc="The output tranches file used by ApplyRecalibration", optional=false)
-    private String TRANCHES_FILE;
+    private File TRANCHES_FILE; // not GATKPath since this name must be accessible to R code
```

Comment:
No way around this?

Reply:
Maybe, but I didn't want to address that in this PR since I think there are other places where we have this same issue. So I'll add fixing the R code as a bullet point in the [uri task](https://github.com/broadinstitute/gatk/issues/6610) ticket.